qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -771,7 +771,7 @@ get_mirror_journal_position()
     can_execute "${cluster}" || return 0
     can_execute "${remote_cluster}" || return 0
     local mirror_uuid=$(rbd mirror pool info --cluster ${cluster} --pool ${pool} --format xml | \
-      xmlstarlet sel -t -v "//peers/peer[site_name='${remote_cluster}']/mirror_uuid")
+      xmlstarlet sel -t -v "//peers/peer[site_name='${remote_cluster}${PEER_CLUSTER_SUFFIX}']/mirror_uuid")
```

Comment:
If it indeed fixes the issue and not this PR specific I suggest opening a tracker ticket for this and pushing as a separate PR, so we could also backport.

Reply:
Ah actually this is something I added here for multipeer so it explains why it was working on master :D.